File: PrintTable.cpp
#include<iostream>
using namespace std;

class DisplayTable
{
    public:
      int No;

      DisplayTable(int iValue)
      {
          No = iValue;
      }

      void PrintTable()
      {
          int iCnt = 0;
          
          cout<<"Table of "<<No<<" is: ";
          for(iCnt = 1; iCnt <=10; iCnt++)
          {
              cout<<"\n"<<No * iCnt;
          }
      }

};

int main()
{
    int iNo = 0;

    cout<<"Enter the number to print table: \n";
    cin>>iNo;

    DisplayTable dobj(iNo);

    dobj.PrintTable();

    return 0;
}

File: PrintTable.java
import java.util.*;

class DisplayTable
{
    public void Table(int No)
    {
        int iCnt = 0;

        System.out.println("Table of "+No+" is:");
        for(iCnt = 1; iCnt <= 10; iCnt++)
        {
            System.out.println(No*iCnt);
        }
    }
}

class PrintTable
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        int iNo = 0;

        System.out.println("Enter the number to print table: ");
        iNo = sobj.nextInt();

        DisplayTable dobj = new DisplayTable();

        dobj.Table(iNo);

    }
}